packages/royalties/contracts/interfaces/IMultiReceiverRoyaltyOverrideCore.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: manifold.xyz
+
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import {
+    IRoyaltySplitter,
+    Recipient
+} from "@manifoldxyz/royalty-registry-solidity/contracts/overrides/IRoyaltySplitter.sol";
+
+/**
+ * Multi-receiver EIP2981 reference override implementation
+ */
+interface IMultiReceiverRoyaltyOverrideCore is IERC165 {
```

Comment:
why doesn't IMultiReceiverRoyaltyOverrideCore extend IMultiReceiverRoyaltyOverride

Reply:
@rishabh0x00 to review

Reply:
@rishabh0x00